workflow/exp/exp.rt_jet_ens_conus12km

```diff
@@ -0,0 +1,91 @@
+#!/user/bin/env bash
+# Aloha! This file is to setup some top level experiment options
+
+export NET=rrfs                    # rrfs, rtma, 
+export MESH_NAME=conus12km         # conus12km, conus3km, atl3km, etc
+export WGF="enkf"   # working group function = "det", "enkf", "ensf", "firewx"
+export EXP_NAME=rrfs${WGF}
+export VERSION=v2.0.3
+export TAG=c3v203
+export OPSROOT=/lfs5/BMC/nrtrr/NCO_dirs/${VERSION}
+export EXPDIR=${OPSROOT}/${EXP_NAME}
+export COMROOT=${OPSROOT}/com      # task input and output data as well as logs
+export DATAROOT=${OPSROOT}/stmp    # task workdirs ($DATA) which to be removed immediately upon task completion unless KEEPDATA=yes
+#
+export REALTIME=true     # true, false
+#
+export DO_ENSEMBLE=true
+export DO_DETERMINISTIC=false
+export ENS_SIZE=30
+#
+export PHYSICS_SUITE="convection_permitting"   # mesoscale_reference, convection_permitting
+export ZETA_LEVELS="L60.txt"
+#
+export DA_IODA="true"
+export DA_JEDI="true"
+export OBSPATH="local.setup"
+
+export DO_CYC="true"
+export CYC_INTERVAL=3
+export FCST_LENGTH=3
+for i in {0..23};    do arr[$i]="03"; done # 3hr fcst
+for i in {0..23..6}; do arr[$i]="06"; done # 3hr fcst every 6hrs
+export FCST_LEN_HRS_CYCLES="${arr[*]}"
+export HISTORY_INTERVAL=1
+export RESTART_INTERVAL=61
+export MPASSIT_GROUP_TOTAL_NUM=1
+export UPP_GROUP_TOTAL_NUM=1
+
+export IC_EXTRN_MDL_NAME="GEFS"
+export IC_OFFSET=36
+export IC_EXTRN_MDL_BASEDIR="/mnt/lfs5/BMC/public/data/grids/gens"
+export IC_EXTRN_MDL_NAME_PATTERN="gep#gmem#/@y@j@H000fHHH"
+export IC_EXTRN_MDL_NAME_PATTERN_B="pgrb2b/gep#gmem#/@y@j@H000fHHH"
```

Comment:
The real-time GEFS data under /mnt/lfs5/BMC/public/data/grids/gens/pgrb2b are already combined grib2 files. Therefore we should have this only:
export IC_EXTRN_MDL_NAME_PATTERN="pgrb2b/gep#gmem#/@y@j@H000fHHH"

Reply:
Thanks, @chunhuazhou 
I think I did not make those changes to this file.
Maybe some of Ming and/or your local changes are not committed to the repo.

Could you create a PR to make the `exp.rt_jet_ens_conus12km` file work as expected? Thanks!